app/jobs/sync_all_users_job.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class SyncAllUsersJob < CronJob
+  self.cron_expression = "0 0 * * *"
```

Comment:
Is this saying run the job at midnight each day?

Reply:
Yep, that is exactly what it is saying :)